lib/screens/home/screens/calendar_screen.dart

```diff
@@ -31,8 +33,15 @@ class CalendarScreen extends ConsumerWidget {
         }
         return Scaffold(
           floatingActionButton: FloatingActionButton.extended(
-            onPressed: () {
-              'Add event'.log();
+            onPressed: () async {
+              final String? url = registrationLink.value?.registrationLink;
+              if (url != null) {
+                if (await canLaunch(url)) {
+                  await launch(url);
+                } else {
+                  print('Could not launch $url');
```

Comment:
Instead of print we could toast smth to let user know that url is not working/say that he can search dzien wydzialu on fb or any other platform

Reply:
Maybe we can use snackbar here @Misiek22113